mail/local/mail.go

```diff
@@ -0,0 +1,183 @@
+package local
```

Comment:
Another _local_ library... this time for making mailboxes. Eventually I'll add a repo here for an offline outbox.